app/routes/project-items.js

```diff
@@ -0,0 +1,148 @@
+  return {
+    backLink,
+    ...(errorSummary ? { errorText: errorSummary } : {}),
+    checkboxesInfrastucture: {
+      idPrefix: 'projectInfrastucture',
+      name: 'projectInfrastucture',
+      fieldset: {
+        legend: {
+          text: 'Reservoir construction and infrastructure',
+          isPageHeading: false,
+          classes: 'govuk-fieldset__legend--m'
+        }
+      },
+      items: setLabelData(
+        projectInfrastucture,
+        ['Construction of dam walls',
+          'Construction of reservoir',
+          'Overflow/spillway',
+          'Synthetic liner',
+          'Abstraction point including pump',
+          'Engineer fees (construction engineers only)',
+          'Fencing for synthetically lined reservoir',
+          'Filtration equipment',
+          'Irrigation pump(s) and controls',
+          'Pipework to fill the reservoir',
+          'Pumphouse',
+          'Underground water distribution main and hydrants',
+          'Electricity installation for pump house',
+          'Water meter'
+        ]
+      ),
+      ...(errorMessage ? { errorMessage: { text: errorMessage } } : {})
+    },
+    checkboxesEquipment: {
+      idPrefix: 'projectEquipment',
+      name: 'projectEquipment',
+      fieldset: {
+        legend: {
+          text: 'Irrigation equipment',
+          isPageHeading: false,
+          classes: 'govuk-fieldset__legend--m'
+        }
+      },
+      items: setLabelData(
+        projectEquipment,
+        ['Boom',
+          'Trickle',
+          'Ebb and flood or capillary bed',
+          'Sprinklers and mist'
+        ]
+      ),
+      ...(errorMessage ? { errorMessage: { text: errorMessage } } : {})
+    },
+    checkboxesTechnology: {
+      idPrefix: 'projectTechnology',
+      name: 'projectTechnology',
+      fieldset: {
+        legend: {
+          text: 'Technology',
+          isPageHeading: false,
+          classes: 'govuk-fieldset__legend--m'
+        }
+      },
+      items: setLabelData(
+        projectTechnology,
+        ['Software to monitor soil moisture levels and schedule irrigation',
+          'Software and sensors to optimise water application'
+        ]
+      ),
+      ...(errorMessage ? { errorMessage: { text: errorMessage } } : {})
+    }
+  }
+}
+
+module.exports = [
+  {
+    method: 'GET',
+    path: '/project-items',
+    handler: (request, h) => {
+      const landOwnership = request.yar.get('landOwnership') || null
+      const backUrl = landOwnership === 'No' ? '/answers' : '/tenancy'
+
+      const projectInfrastucture = request.yar.get('projectInfrastucture') || null
+      const projectEquipment = request.yar.get('projectEquipment') || null
+      const projectTechnology = request.yar.get('projectTechnology') || null
+
+      return h.view(
+        'project-items',
+        createModel(null, null, backUrl, projectInfrastucture, projectEquipment, projectTechnology)
```

Comment:
we do not need errorSummary argument here it should be **createModel(null, null, backUrl, projectInfrastucture, projectEquipment, projectTechnology)**